### PR TITLE
[schema] Add twitter schema file #936

### PR DESCRIPTION
**Pull request by VSevagen**
Signed-off-by: sevagenv <sevagenv@gmail.com>

This is a PR for issue #936. It is my very first PR so there might be a few issues with it. Looking forward to get it reviewed.

### CODE REVIEW

**Review by sduenas — CHANGES_REQUESTED** [review]
Thanks for the PR. Would you mind to check #964? There's some fields that are deprecated and shouldn't be part of the schema.

Also, the commit message should be:

```
[schema] Add twitter schema file

Twitter schema was missing. This commit serves to add it.

Signed-off-by: sevagenv <sevagenv@gmail.com>
```

File: schema/twitter.csv [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+author_uuid,keyword,true,"Author UUID from SortingHat."
+created_at,date,true,"Date when tweet was created."
+geolocation,geo_point,true,N\A
+grimoire_creation_date,date,true,"Message creation date"
+hashtags_analyzed,keyword,true,"Text of hashtag."
+id,long,true,"Unique identifier of the tweet."
+id_str, keyword,true,"Unique identifier of tweet in string format."
+in_reply_to_screen_name,keyword,true,"Screen name of original tweet author in case tweet is a reply."
+in_reply_to_user_id_str,keyword,true,"ID of parent Tweet's author in case tweet is a reply."
+is_twitter_twitter,long,true,N\A
```
Comment (sduenas) [inline schema/twitter.csv:19]: This field is `is_twitter_tweet`, it's used to count number of tweets in complex agregations.

File: schema/twitter.csv [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+place.bounding_box.coordinates,long,keyword,"A series of longitude and latitude points, defining a box which will contain the Place entity this bounding box is related to"
+place.bounding_box.type,keyword,true,"A box of coordinates"
+place.country,keyword,true,"Name of the country containing this place"
+place.country_code,keyword,true,"Shortened country code representing the country containing this place."
+place.full_name,keyword,true,"Full human-readable representation of the place’s name."
+place.id,keyword,true,"ID representing this place"
+place.name,keyword,true,"Short human-readable representation of the place’s name."
+place.place_type,keyword,true,"The type of location represented by this place"
+place.url,keyword,true,"URL representing the location of additional place metadata for this place."
+project,keyword,true,"Project."
```
Comment (sduenas) [inline schema/twitter.csv:35]: Project name.

File: schema/twitter.csv [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+project_1,keyword,true,"Project (if more than one level is allowed in project hierarchy)."
+retweet_count,long,true,"Number of times tweet has been retweeted."
+retweeted,long,true,"Indicates whether this Tweet has been Retweeted by the authenticating user."
+tag,keyword,true,"The text of the Hashtag."
```
Comment (sduenas) [inline schema/twitter.csv:39]: `tag` is the tag set by perceval when retrieving the item.

**Review by sduenas — CHANGES_REQUESTED** [review]
The PR looks ok. Although there are minor changes you need to address before merging it.

File: schema/twitter.csv [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+author_uuid,keyword,true,"Author UUID from SortingHat."
+created_at,date,true,"Date when tweet was created."
+geolocation,geo_point,true,N\A
+grimoire_creation_date,date,true,"Message creation date"
+hashtags_analyzed,keyword,true,"Text of hashtag."
+id,long,true,"Unique identifier of the tweet."
+id_str, keyword,true,"Unique identifier of tweet in string format."
+in_reply_to_screen_name,keyword,true,"Screen name of original tweet author in case tweet is a reply."
+in_reply_to_user_id_str,keyword,true,"ID of parent Tweet's author in case tweet is a reply."
+is_twitter_twitter,long,true,"Count number of tweets in complex aggregations."
```
Comment (sduenas) [inline schema/twitter.csv:19]: I think the name of the field is `is_twitter_tweet`, am I wrong?

Comment (VSevagen) [reply schema/twitter.csv:19]: I'm not sure. I'll check once

Comment (VSevagen) [reply schema/twitter.csv:19]: In the twitter index pattern, its listed as `is_twitter_twitter`.

Comment (sduenas) [reply schema/twitter.csv:19]: The index patter is wrong too. The real field is `is_twitter_tweet`. This is from a real example.
imagen

I suggest to change it to the name of this field and to update the index pattern too. Do you mind to change it too? It's in `grimoirelab-sigils` repository.

Comment (VSevagen) [reply schema/twitter.csv:19]: Alright. I'll change it. I'll create another PR for updating the index pattern.

**Review by sduenas — APPROVED** [review]
LGTM

**Pull request merged**